test/acceptance/features/steps/steps.py

```diff
@@ -301,26 +302,37 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
     context.application = application
     context.application_type = "knative"
 
+def get_sbr_secret(context):
+    openshift = Openshift()
+    time.sleep(10)
+    secret = ""
+    if "sbr_name" in context:
+        json_path = "{.status.secret}"
+        secret = polling2.poll(lambda:
+            openshift.get_resource_info_by_jsonpath("servicebindings", context.sbr_name, context.namespace.name, json_path), step=5, timeout=800,
+                ignore_exceptions=(ValueError,))
+        assert secret is not None, f"Invalid result for SBO jsonpath: {secret}."
```

Comment:
we should not return from polling only when the result is not None.

Reply:
e7384ea